data/core/units/saurians/Javelineer.cfg

```diff
@@ -0,0 +1,95 @@
+#textdomain wesnoth-units
+[unit_type]
+    id=Saurian Javelineer
+    name= _ "female^Saurian Javelineer"
+    race=lizard
+    gender=female
+    image="units/saurians/javelineer/javelineer.png"
+    profile="portraits/saurians/skirmisher.webp"
+    hitpoints=50
+    movement_type=lizard
+    movement=7
+    experience=150
+    level=3
+    alignment=chaotic
+    advances_to=null
+    {AMLA_DEFAULT}
+    cost=52
+    usage=archer
+    description= _ "After years of combat practice, saurians warriors that are the most talented in the skill of spear throwing might advance to the ranks of the javelineers. With better equipment and exceptional aim, saurian javelineers are not to be underestimated."
```

Comment:
I understand this is just a place-holder, but is 'saurians warriors' supposed to be 'saurian warriors'?

Reply:
That's a typo, although the entire text is a placeholder.